dbt/include/trino/macros/materializations/seeds/helpers.sql

```diff
@@ -0,0 +1,85 @@
+{% macro trino__get_batch_size() %}
+  {{ return(1000) }}
+{% endmacro %}
+
+
+{% macro create_bindings(row, types) %}
+  {% set values = [] %}
+  {% set re = modules.re %}
+  
+  {%- for item in row -%}
+      {%- set type = types[loop.index0] -%}
+      {%- set match_type = re.match("(\w+)(\(.*\))?", type) -%}
+      {%- if item is not none and item is string and 'interval' in match_type.group(1) -%}
+        {%- do values.append((none, match_type.group(1).upper() ~ " " ~ item)) -%}
+      {%- elif item is not none and item is string and 'varchar' not in type.lower() -%}
+        {%- do values.append((none, match_type.group(1).upper() ~ " '" ~ item ~ "'")) -%}
+      {%- elif item is not none and 'varchar' in type.lower() -%}
+        {%- do values.append((get_binding_char(), item|string())) -%}
+      {%- else -%}
+        {%- do values.append((get_binding_char(), item)) -%}
+      {% endif -%}
+  {%- endfor -%}
+  {{ return(values) }}
+{% endmacro %}
+
+
+{# 
+  We need to override the default__load_csv_rows macro as Trino requires values to be typed according to the column type
+  as in following example:
+  
+  create table "memory"."default"."data_date_trunc" ("id" INTEGER,"user_id" INTEGER,"order_date" DATE,"status" VARCHAR)
```

Comment:
There is a little inconsistency between the columns in `CREATE TABLE` and `INSERT` statement in this comment.